crypto/ocsp/ocsp_client.c

```diff
@@ -91,14 +103,12 @@ int OCSP_single_get0_status(OCSP_SINGLERESP *single, int *reason,
       }
     }
   }
-  /*
-   * Look up when certificate was last updated and when is next update time
-   */
+  // Look up when certificate was last updated and when is next update time
   if (thisupd != NULL) {
-    *thisupd = single->thisUpdate;
+    OPENSSL_memcpy(thisupd, &single->thisUpdate, sizeof(&single->thisUpdate));
   }
   if (nextupd != NULL) {
-    *nextupd = single->nextUpdate;
+    OPENSSL_memcpy(nextupd, &single->nextUpdate, sizeof(&single->nextUpdate));
```

Comment:
same comment as above.

Reply:
Purpose of retrieving `nextupd ` is to extract the time value of when the certificate should be updated again from the OCSP responder. The `nextupd ` value is extracted only to check if the certificate timestamp is valid and is not intended to be changed in another function.
Changing back for now.